src/main/java/core/basesyntax/impl/ReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.util.Reader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class ReaderImpl implements Reader {
```

Comment:
It is not mandatory, but it is more common to use the `Service` in the names
```suggestion
public class ReaderServiceImpl implements ReaderService {
```